### ISSUE TITLE
Scrolling issues in FireFox and IE11.

### ISSUE THREAD

**Issue by Andrii-A** [idiotWu/smooth-scrollbar#48]
Hi, I found an issue with the scrollbar in FireFox and IE11.

When you scroll the page with a mouse (by dragging handler up/down) you get highlighted all items on the page if a cursor doesn't stay inside the handler.

## Expected Behavior
Would be great to have the same consistent behavior, as we have in Chrome, in FireFox and IE11 as well.

## Steps to Reproduce
Open the example in FireFox or IE11:
http://idiotwu.github.io/smooth-scrollbar/

Click on handler and drag it up/down (don't release mouse).
if you move the cursor outside of the handler (without releasing mouse button), you will get highlighted items on the page.

Screenshot attached:
![scrollbar issue](https://cloud.githubusercontent.com/assets/15634416/20689096/a5364f00-b591-11e6-9ba2-675a78f7d8c8.jpg)

**Comment by idiotWu** [comment]
Thanks for reporting. This issue will be fixed in the next release `8.0.0` :)

**Comment by idiotWu** [comment]
I am busy with the other project these days, so I'm afraid v8 won't come so soon. Anyone who wants to fix this issue, simply do:

```css
.scrollbar-track {
     user-select: none;
}
```

**Comment by Jerek0** [comment]
It seems the issue's still present on Safari 10.0 😕

**Comment by kimwes** [comment]
The suggested fix didn't work for me. Here's how I solved it. I'm using jQuery here but one can do it with native js easily. First I had to add some way of detecting that scrollbar is being dragged. I tried to hook to a dragstart event that I saw being invoked in the drag.js but apparently I called it wrong so I hooked on the general mousedown.
`            $(".scrollbar-track").on('mousedown', function(event) {
                container.addClass("dragging");
            });
            $(window).on('mouseup', function(event) {
                container.removeClass("dragging");
            });
`
Then the css that was copied from the solution above with vendor prefixes.
`.dragging {
    -o-user-select: none;
    -ms-user-select: none;
    -moz-user-select: none;
    -webkit-user-select: none;
    user-select: none;
}`
It would be nice if the library added the dragging or similar class itself and took care of the css.
